### PR TITLE
Add Maker Party 2015 page

### LINKED ISSUE #1000: Implement /activities/maker-party-2015
This is the page that will be linked to from /events

This page should follow the same template as /activities/web-lit-basics/ and /activities/protect-your-data/

The content can be found starting on line 21 of this pad, but needs to be finalized: https://teach.etherpad.mozilla.org/teachsite-eventspage

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #1000.

### CODE REVIEW

File: less/components/sidebar.less [hunk 1]
```diff
@@ -14,36 +14,25 @@ html.no-js .sidebar {
...
   }
-  &.web-lit-basics {
-    .sidebarColorizer(@pinkSidebarColor);
-  }
-  &.protect-your-data {
-    .sidebarColorizer(@pinkSidebarColor);
-  }
-  &.web-literacy {
-    .sidebarColorizer(@greenSidebarColor);
-  }
```
Comment (mmmavis) [inline less/components/sidebar.less]: I added `&.maker-party-2015` and grouped classes that share the same sidebar colour together

File: components/activity-section.jsx [hunk 2]
```diff
@@ -13,25 +13,33 @@ var ActivitySection = React.createClass({
...
-                  </a>
-                  <p>
-                    <em>{activity.subtitle}</em>
-                  </p>
-                  <p className="description" dangerouslySetInnerHTML={{__html: activity.description}}></p>
-                </div>
-              </Illustration>
-            </div>
-          )
+          if (Object.keys(activity).length === 1 && activity.title) {
```
Comment (mmmavis) [inline components/activity-section.jsx]: Adding this to handle cases like this
screen shot 2015-06-18 at 4 04 45 pm

**Comment by mmmavis** [comment]
@toolness this is ready for review :smile_cat: 

just a reminder for myself to update https://github.com/mozilla/teach.webmaker.org/pull/1060/files#r32890076 once this is merged :+1:

File: components/activity-section.jsx (hunk 2 above)
Comment (toolness) [inline components/activity-section.jsx]: Hmm, for these situations, what do you think of changing the `title` property to `placeholderText` or something a bit more semantically meaningful? That could make reading the code a bit more self-explanatory, and also avoid the `Object.keys` length check.

File: less/components/sidebar.less (hunk 1 above)
Comment (toolness) [inline less/components/sidebar.less]: Ah cool, thanks!

**Comment by toolness** [comment]
This looks good to me! Feel free to address my [nit](https://github.com/mozilla/teach.webmaker.org/pull/1048#discussion_r33562926) if you want, is ok to leave it as-is and push too though.

File: components/activity-section.jsx (hunk 2 above)
Comment (mmmavis) [inline components/activity-section.jsx]: brilliant idea! https://github.com/mozilla/teach.webmaker.org/commit/237a205322ab385a832504a8ff213473cb6b0895 :wink:

**Comment by mmmavis** [comment]
i'm holding off on this until https://github.com/mozilla/teach.webmaker.org/pull/1064 gets pushed to PROD first

**Pull request merged**